.github/workflows/check-release.yml

```diff
@@ -17,13 +16,18 @@ jobs:
       image: python:latest
     env:
       GITHUB_PAT: ${{ secrets.INSTALL_GITHUB }}
-    # If the PR is coming from a fork (pull_request_target), ensure it is opened
-    # by "dependabot[bot]". Otherwise, clone it normally.
       CC_API_ORGANISATION: ${{ secrets.CC_API_ORGANISATION }}
       CC_API_KEY: ${{ secrets.CC_API_KEY }}
-    if: |
-        (github.event_name == 'pull_request_target' && github.actor == 'dependabot[bot]') ||
-        (github.event_name != 'pull_request_target' && github.actor != 'dependabot[bot]')
+    if: >
+      (github.event_name != 'pull_request' && !github.event.pull_request.head.repo.fork) ||
+      (github.event_name == 'pull_request' &&
+      (github.event.pull_request.head.repo.fork || github.actor == 'dependabot[bot]'))
+
+    # Add concurrency to ensure only one run at a time per branch
+    concurrency:
+      group: ${{ github.ref }}
```

Comment:
What does '${{ github.ref }}' stand for? What is the effect of using it?

Reply:
Actually, this lines is redundant for our case, dropped